pkg/util/utils_test.go

```diff
@@ -250,6 +250,13 @@ func TestMaskProxyPassword(t *testing.T) {
 		{
 			input:  "https://fr@ank5t3in:an0th3erF@akeP@55word@minikube.sigs.k8s.io",
 			output: "https://fr@ank5t3in:*****@minikube.sigs.k8s.io",
+		}, {
+			input:  "http://aaa:bbb@internet-app.corp.thales:8080",
```

Comment:
@ComradeProgrammer I prefer we use psudo names not actual company names here, how about changing thales to something like "abcdcompany"

Reply:
done